### ISSUE TITLE
add more tests

### ISSUE THREAD

**Issue by leonorader** [floppylab/markdown2document#12]
Right now, the lib has only a few tests, I tested only manually.

**Comment by miguelhrocha** [comment]
Do you want help with this @leonorader ?

**Comment by leonorader** [comment]
@aComanche it would be great if you would help with this! :)

**Comment by miguelhrocha** [comment]
Do you have some any priority list to see what can I test first?

**Comment by leonorader** [comment]
Not really, I was about to start with unit tests, but it is really up to you :) @aComanche
